Updating README file

Bhavneet-Sharma opened the pull request:
# Description
Updating README file

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [x] I have performed Ansible Sanity test using --docker default
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Unit test
- [X] Functional Test

forrestxia commented on .github/workflows/ansible-test.yml:
```diff
@@ -48,19 +49,16 @@ jobs:
       matrix:
-        python: ['3.9', '3.10', '3.11']
+        python: ['3.10', '3.11']
         ansible:
```
Is 3.12 needed for the matrix run?

Bhavneet-Sharma replied:
we can try for python 3.12 as well. Will update.

Bhavneet-Sharma replied:
Done,

forrestxia commented on README.md:
```diff
@@ -35,7 +35,7 @@ The capabilities of the Ansible modules are managing SDCs, volumes, snapshots, s
 |---------------------|-----------------------|-------|--------------------|--------------------------|
-| v2.6.0 |3.6 <br> 4.0 <br> 4.5 <br> APEX Block Storage for Mircrosoft Azure <br> APEX Block Storage for AWS | 1.14.0 | 3.9.x <br> 3.10.x <br> 3.11.x <br> 3.12.x | 2.15 <br> 2.16 <br> 2.17 |
+| v2.6.0 |4.0 <br> 4.5 <br> 4.6 <br> APEX Block Storage for Mircrosoft Azure <br> APEX Block Storage for AWS | 1.14.0 | 3.9.x <br> 3.10.x <br> 3.11.x <br> 3.12.x | 2.15 <br> 2.16 <br> 2.17 |
 
```
Add Ansible 2.18 as well?

Bhavneet-Sharma replied:
Done,

forrestxia commented on .github/workflows/ansible-test.yml:
```diff
@@ -83,19 +85,16 @@ jobs:
       matrix:
-        python: ['3.9', '3.10', '3.11']
+        python: ['3.10', '3.11']
         ansible:
```
3.12 on line 88 as well?

Bhavneet-Sharma replied:
Done.

Bhavneet-Sharma commented:
> @Bhavneet-Sharma is there a need to update pypowerflex dependency to use an explicit version string in requirements.txt?

@forrestxia , we're good with implicit implementation, As have a check for minimum version in code as _min_ver = '1.14.1'_ which support all functionalities we support.

forrestxia commented:
> > @Bhavneet-Sharma for any python lib dependency, let's use the explicit version string starting from now, we've updated ansible-powerstore to use explicit version of pypowerstore.
> 
> @forrestxia I have made the changes. Please review.

@Bhavneet-Sharma does it follow [the PEP 440 version specifiers](https://peps.python.org/pep-0440/#version-specifiers)?

Bhavneet-Sharma commented:
> > > @Bhavneet-Sharma for any python lib dependency, let's use the explicit version string starting from now, we've updated ansible-powerstore to use explicit version of pypowerstore.
> > 
> > 
> > @forrestxia I have made the changes. Please review.
> 
> @Bhavneet-Sharma does it follow [the PEP 440 version specifiers](https://peps.python.org/pep-0440/#version-specifiers)?

Yes, now it follows the PEP versioning. There was typo, I have fixed. Thanks for comment. 
please review.
